app/Http/Controllers/Root/RootController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Http\Controllers\Root;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+
+/**
+ * RootController
+ *
+ * タブバーやサイドメニューなど動線管理を行う
+ *
+ */
+class RootController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition